db/seeds.rb

```diff
@@ -6,7 +6,11 @@
 FactoryGirl.create(:skill, name: "skill2", featured: true)
 FactoryGirl.create(:skill, name: "skill2", featured: false)
 
+puts " Successfully added skills from seed data. ".center(70, '~*~')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.